client/state/login/selectors.js

```diff
@@ -268,7 +260,7 @@ export const getSocialAccountLinkEmail = ( state ) =>
  * @returns {?string}         service name that is being linked
  */
 export const getSocialAccountLinkService = ( state ) =>
-	get( state, 'login.socialAccountLink.authInfo.service', null );
+	state.login.socialAccountLink?.authInfo?.service ?? null;
```

Comment:
The optional chain op after `socialAccountLink` is not needed, the rest is OK.

Reply:
Done.